qa/purchase_offline_error.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import json
+import time
+from collections import OrderedDict
+from test_framework.test_framework import OpenBazaarTestFramework, TestFailure
+
+
+class PurchaseOfflineErrorTest(OpenBazaarTestFramework):
+
+    def __init__(self):
+        super().__init__()
+        self.num_nodes = 3
+
+    def run_test(self):
+        alice = self.nodes[0]
+        bob = self.nodes[1]
+
+        # post profile for alice
+        with open('testdata/profile.json') as profile_file:
+            profile_json = json.load(profile_file, object_pairs_hook=OrderedDict)
+        api_url = alice["gateway_url"] + "ob/profile"
+        requests.post(api_url, data=json.dumps(profile_json, indent=4))
+
+        # post listing to alice
+        with open('testdata/listing.json') as listing_file:
+            listing_json = json.load(listing_file, object_pairs_hook=OrderedDict)
+        if self.bitcoincash:
+            listing_json["metadata"]["pricingCurrency"] = "tbch"
```

Comment:
Would love these o be extracted into `test_framework` to work like:
`listing_json = self.return_bitcoin_listing(bch=true)`
This could be done throughout our QA tests.

Reply:
Good suggestion